tags_add/verify.py

```diff
@@ -0,0 +1,54 @@
+from git import TagObject
+from git_autograder.core import GitAutograderExercise, GitAutograderStatus
+
+FIRST_TAG = "first-pilot"
+SECOND_TAG = "v1.0"
+SECOND_TAG_MSG = "first full duty roster"
+MARCH_MSG_FRAGMENT = "Update roster for March"
+
+def verify(exercise: GitAutograderExercise):
+    repo = exercise.repo.repo
+
+    root_sha = next(repo.iter_commits(rev="HEAD", reverse=True)).hexsha
+
+    march_commit = None
+    for c in repo.iter_commits("HEAD"):
+        if MARCH_MSG_FRAGMENT in c.message:
+            march_commit = c
+            break
```

Comment:
Our general autograding philosophy is to grade the tasks in order. So if you don't need this variable till task 2, I would suggest moving it down to later on.